server/lib/device/device.getDeviceFeaturesStates.js

```diff
@@ -0,0 +1,73 @@
+const { Op } = require('sequelize');
+const db = require('../../models');
+const { NotFoundError } = require('../../utils/coreErrors');
+
+const DEFAULT_OPTIONS = {
+  skip: 0,
+  order_dir: 'ASC',
+  order_by: 'created_at',
+};
+
+/**
+ * @description Get all features states aggregates.
+ * @param {string} selector - Device selector.
+ * @param {Object} [options] - Options of the query.
+ * @param {string} [options.from] - Start date in UTC format "yyyy-mm-ddThh:mm:ss:sssZ"
```

Comment:
I think this parameter is not optional, so shouldn't be between brackets []

Reply:
Fixed